pkgs/by-name/cm/cmd-unwrapped/package.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cmd-wrapped";
```

Comment:
Why is the package in `cmd-unwrapped` then?

Reply:
typo, sorry......